tasks/controllers/user.js

```diff
@@ -42,6 +47,17 @@ export const Controller = {
             } else {
                 response.status(204).send({});
             }
+        },
+        login(request, response) {
+            const user = getUserByLogin(request.body.login);
+            if (user === undefined || user.password !== request.body.password) {
```

Comment:
Consider having `user` instead of `user === underfined` as a truthy value in the if statements,